x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/index.ts

```diff
@@ -4,7 +4,6 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { ExpressionFunctionAST } from '@kbn/interpreter/common';
```

Comment:
As an aside -- At some point y'all should consider removing all of the old `@kbn/interpreter` imports from Lens since you can access the same types (with slightly different names) from expressions.

I just did so in the files I touched here